GNUmakefile

```diff
@@ -1,32 +1,35 @@
 TEST?=./...
 PKG_NAME=pkg/xray
+PKG_VERSION_PATH=github.com/jfrog/terraform-provider-xray/${PKG_NAME}
 #VERSION := $(shell git tag --sort=-creatordate | head -1 | sed  -n 's/v\([0-9]*\).\([0-9]*\).\([0-9]*\)/\1.\2.\3/p')
 # Replace explicit version after the first release
 VERSION := 0.0.0
 NEXT_VERSION := $(shell echo ${VERSION}| awk -F '.' '{print $$1 "." $$2 "." $$3 +1 }' )
+BINARY_NAME=terraform-provider-xray
+BUILD_PATH=terraform.d/plugins/registry.terraform.io/jfrog/xray/${NEXT_VERSION}/darwin_amd64
```

Comment:
this only makes sense on mac. If you're going to do this you should figure out how to get it right on all platforms

Reply:
I know. @alexhung you have the same darwin release. RT project has only darwin as well. GH actions generate all types, do we need to support all that in our makeFile?

Reply:
It's for localized builds

Reply:
This change didn't change any behavior. It was just me extracting the same path into a script variable. So if there's an issue and we need to change this, go ahead.

Reply:
I'll leave it for the future PR